extensions/WEBGL_compressed_texture_pvrtc/extension.xml

```diff
@@ -13,6 +13,13 @@
     <api version="1.0"/>
   </depends>
   <overview>
+    <p>
+      <b>Note: this extension is deprecated.</b> On systems that support this compressed texture
+      format, please consider using the <a
+      href="../WEBGL_compressed_texture_etc">WEBGL_compressed_texture_etc</a> or <a
+      href="WEBGL_compressed_texture_astc">WEBGL_compressed_texture_astc</a> formats instead. They
```

Comment:
Should the ASTC link start with `../`?

Reply:
Yes - thanks for catching that. Fixed, and tested the built HTML locally.